Formula/terraform-docs.rb

```diff
@@ -26,16 +26,12 @@ class TerraformDocs < Formula
         :revision => "c8470e45692f168e8b380c5d625327e756d7d0a9"
   end
 
-  def version
-    stable.url.split("/").last.gsub(/\.tar\.gz/, "")
-  end
-
   def install
     ENV["GOPATH"] = buildpath
     (buildpath/"src/github.com/segmentio/terraform-docs").install buildpath.children
     Language::Go.stage_deps resources, buildpath/"src"
     cd "src/github.com/segmentio/terraform-docs" do
-      system "go build -o #{bin}/terraform-docs -ldflags '-X main.version=#{version}'"
+      system "go", "build", "-o", "#{bin}/terraform-docs", "-ldflags", "-X main.version=#{stable.version}"
```

Comment:
I'd just do `version` instead of `stable.version` since we don't want `--HEAD` masquerading as a release version. Also, it shouldn't go passed the eightieth column, so
```
      system "go", "build", "-o", "#{bin}/terraform-docs", "-ldflags,
             "-X main.version=#{version}"
```